### ISSUE TITLE
Not working under Ubuntu 16.04.3 LTS

### ISSUE THREAD

**Issue by Zuza** [aveferrum/wine-starcraft-installer#4]
tl;dr: I just downloaded and installed it. Any idea what might be causing the issues listed below? I would imagine my setup doesn't differentiate that much from others.

**(1) [minor]** the desktop icons never appeared on the desktop

**(2) [medium]** when manually running the Starcraft.desktop, I get the following error.

> wine: cannot find L"C:\\windows\\system32\\StarCraft.exe"
> ./StarCraft.desktop: line 6: StarCraft: command not found
> ./StarCraft.desktop: line 7: syntax error near unexpected token `('
> ./StarCraft.desktop: line 7: `Path=/home/zuza/.wine-starcraft-ed/sc-prefix/drive_c/Program Files (x86)/StarCraft'

**(3) [critical]** when manually running Starcraft.exe, I get the following errors:
Run command:
> env WINEPREFIX="/home/zuza/.wine-starcraft-ed/sc-prefix" /home/zuza/.wine-starcraft-ed/wine-staging/bin/wine ~/.wine-starcraft-ed/sc-prefix/drive_c/Program\ Files\ \(x86\)/StarCraft/Starcraft.exe

Output:

> fixme:winediag:start_process Wine Staging 2.14 is a testing version containing experimental patches.
> fixme:winediag:start_process Please mention your exact version when filing bug reports on winehq.org.
> fixme:actctx:parse_requested_execution_level_elem unknown attr L"xmlns:ms_asmv3"=L"urn:schemas-microsoft-com:asm.v3"
> fixme:actctx:parse_requested_execution_level_elem unknown attr L"ms_asmv3:uiAccess"=L"false"
> Wine cannot find the ncurses library (libncursesw.so.6).
> fixme:thread:create_user_shared_data_thread Creating user shared data update thread.
> fixme:module:load_library unsupported flag(s) used (flags: 0x00000800)
> fixme:module:load_library unsupported flag(s) used (flags: 0x00000800)
> fixme:module:load_library unsupported flag(s) used (flags: 0x00000800)
> fixme:module:load_library unsupported flag(s) used (flags: 0x00000800)
> fixme:module:load_library unsupported flag(s) used (flags: 0x00000800)
> err:module:attach_process_dlls "ClientSdk.dll" failed to initialize, aborting
> err:module:LdrInitializeThunk Main exe initialization for L"C:\\Program Files (x86)\\StarCraft\\Starcraft.exe" failed, status c0000094

comment: I assume that the ClientSdk.dll failure is causing the problem.

------------------------------------ **Platform Info** ------------------------------------------------------------

I should have a (standard) installation of Ubuntu 16. Not sure why would it not work for me if it works for others.

Kernel (uname -a): Linux my-laptop 4.4.0-98-generic #121-Ubuntu SMP Tue Oct 10 14:24:03 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux
Distro: Ubuntu 16.04.3 LTS
Wine version: wine-2.14 (Staging)
Starcraft version: the latest from us.battle.net: StarCraft® Anthology (1.20.x)

**Comment by edemirkan** [comment]
Hi @Zuza I don't know if you still have the problem but if you're interested please give another try as the script has recently updated.

**Comment by Zuza** [comment]
I changed my laptop/configuration, but it might get interesting again in the future :) Thanks
